pkg/virt-controller/watch/drain/disruptionbudget/disruptionbudget.go

```diff
@@ -496,15 +518,19 @@ func (c *DisruptionBudgetController) sync(key string, vmiExists bool, vmi *virtv
 			// pdb for non existent vmi
 			log.Log.Object(vmi).Infof("deleting pdb %s/%s due to VMI not existing anymore", pdb.Namespace, pdb.Name)
 			return c.deletePDB(key, pdb, vmi)
+		} else if isPDBFromOldController(pdb) {
+			// pdb coming from an old migration controller
+			log.Log.Object(vmi).Infof("deleting pdb %s/%s generated by an old migration controller", pdb.Namespace, pdb.Name)
+			return c.deletePDB(key, pdb, vmi)
```

Comment:
I would cover this at least by a unit test.

Reply:
good point! added.